core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/instruction/data/text/TextSubstring.java

```diff
@@ -48,7 +47,7 @@ public String generateOutput(String parameters) {
                 start += text.length()+1;
 
             verifyArguments(text, start);
-            return text.substring(start);
+            return text.replaceAll("\r\n|\r|\n","").substring(start);
```

Comment:
you cannot remove the newlines in the text being 'substringed'. There is a possibility the 'substringed' text contains a newline